operator/pkg/model/translation/envoy_http_connection_manager.go

```diff
@@ -82,5 +82,6 @@ func (i *cecTranslator) desiredHTTPConnectionManager(name, routeName string) (ci
 		connectionManager = fn(connectionManager)
 	}
 
-	return toXdsResource(connectionManager, envoy.HttpConnectionManagerTypeURL)
+	res, _ := toXdsResource(connectionManager, envoy.HttpConnectionManagerTypeURL)
+	return res
```

Comment:
Is this really a good idea? Ignoring error results smells funny to me.

Reply:
it's unlikely we have unmarshal error here as the object is constructed from go struct, however, thinking about it again, it's probably better to return error and bubble it up.